### PR TITLE
Release Datadog::Monitors::Monitor 1.0.2

### PR DESCRIPTION
**Pull request by bkabrda**
### What does this PR do?

Release version 1.0.2 of Datadog::Monitors::Monitor

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: datadog-monitors-monitor-handler/CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # Changelog
 
+# 1.0.2
+
+Released on 2019-11-21
+
+[Link to Resource] s3://datadog-cloudformation-resources/datadog-monitors-monitor/datadog-monitors-monitor-1.0.2.zip
+[BUGFIX] Use `Double` for all previously `Float` values to get sufficient precision for high thresholds.
```
Comment (nmuesch) [inline datadog-monitors-monitor-handler/CHANGELOG.md]: Is is just thresholds ?

Comment (bkabrda) [reply datadog-monitors-monitor-handler/CHANGELOG.md]: Yeah. Feel free to see the commit: https://github.com/DataDog/datadog-cloudformation-resources/commit/0e31ad85cfbd8cadac2e61f2f908113626623222

Comment (nmuesch) [reply datadog-monitors-monitor-handler/CHANGELOG.md]: Ahh ok, sorry. When I read I assumed Critical threshold, not all. LGTM

**Comment by bkabrda** [comment]
Thanks for the reviews, merging.

**Pull request merged**